models/iaf_psc_alpha.h

```diff
@@ -40,71 +40,98 @@ namespace nest
-grid given by the computation step size. Incoming as well as emitted
-spikes are forced to that grid.
+The membrane potential evolves according to
 
-An additional state variable and the corresponding differential
-equation represents a piecewise constant external current.
+.. math::
 
-The general framework for the consistent formulation of systems with
-neuron like dynamics interacting by point events is described in
-[1]_.  A flow chart can be found in [2]_.
+   \frac{dV_m}{dt} = -\frac{V_m - E_L}{\tau_m} + \frac{I_{\text{syn}} + I_e}{C_m}
 
-Critical tests for the formulation of the neuron model are the
-comparisons of simulation results for different computation step
-sizes and the testsuite contains a number of such tests.
+where the synaptic input current :math:`I_{\text{syn}}(t)` is discussed below and :math:`I_e` is
+a constant input current set as a model parameter.
 
-The ``iaf_psc_alpha`` is the standard model used to check the consistency
-of the nest simulation kernel because it is at the same time complex
-enough to exhibit non-trivial dynamics and simple enough compute
-relevant measures analytically.
+A spike is emitted at time step :math:`t^*=t_{k+1}` if
 
-.. note::
+.. math::
+
+   V_m(t_k) < V_{th} \quad\text{and}\quad V_m(t_{k+1})\geq V_{th} \;.
+
+Subsequently,
+
+.. math::
+
+   V_m(t) = V_{\text{reset}} \quad\text{for}\quad t^* \leq t < t^* + t_{\text{ref}} \;,
+
+i.e., the membrane potential is clamped to :math:`V_{\text{reset}}` during the refractory period.
+
+The synaptic input current has an excitatory and an inhibitory component
+
+.. math::
+
+   I_{\text{syn}}(t) = I_{\text{syn, ex}}(t) + I_{\text{syn, in}}(t)
+
+where
+
+.. math::
 
-   The present implementation uses individual variables for the
-   components of the state vector and the non-zero matrix elements of
-   the propagator. Because the propagator is a lower triangular matrix,
-   no full matrix multiplication needs to be carried out and the
-   computation can be done "in place", i.e. no temporary state vector
-   object is required.
+   I_{\text{syn, ex}}(t) = \sum_{(t_m, w_m)\\w_m\geq 0} w_m i_{\text{syn, ex}}(t-t_m)
 
-   The template support of recent C++ compilers enables a more succinct
-   formulation without loss of runtime performance already at minimal
-   optimization levels. A future version of iaf_psc_alpha will probably
-   address the problem of efficient usage of appropriate vector and
-   matrix objects.
+is the sum over all incoming excitatory spikes, with a corresponding expression for inhibitory spikes (:math:`w_m < 0`).
+Finally, individual post-synaptic currents are given by
+
+.. math::
+
+   i_{\text{syn, X}}(t) = \frac{e}{\tau_{\text{syn, X}}} t e^{-\frac{t}{\tau_{\text{syn, X}}}} \Theta(t)
+
+where :math:`\Theta(x)` is the Heaviside step function. The PSCs are normalized to unit maximum, i.e.,
+
+.. math::
+
+   i_{\text{syn, X}}(t) = 1 \Leftrightarrow t = \tau_{\text{syn, X}} \;.
```

Comment:
I don't understand the notation here

Reply:
Since the function has a single global maximum of 1 when t=tau_syn, left-hand side implies right-hand side and vice versa.

Reply:
I changed this line, I hope you find the new version more readable.